api/src/pcapi/routes/backoffice/accounts/blueprint.py

```diff
@@ -351,6 +352,8 @@ def render_public_account_details(
             {
                 "edit_account_form": edit_account_form,
                 "edit_account_dst": url_for(".update_public_account", user_id=user.id),
+                "unvalidation_dst": url_for(".invalidate_public_account_password", user_id=user.id),
+                "unvalidation_form": account_forms.UnvalidationForm(),
```

Comment:
```suggestion
                "invalidation_dst": url_for(".invalidate_public_account_password", user_id=user.id),
                "invalidation_form": account_forms.InvalidationForm(),
```
Mais « invalidation » tout seul n'aide pas à comprendre dans le contexte d'un compte, « ResetPasswordForm » ou quelque chose du genre serait plus clair.

Reply:
Mais ResetPasswordForm suppose un reset du mot de passe alors que c'est plus une invalidation pour forcer l'utilisateur à le reset. Du coup je me demande si : " invalidationPasswordForm " serai plus approprié? qu'en penses tu?

Reply:
« invalidate password » ou « password invalidation »